LoopKit/TherapySetting.swift

```diff
@@ -80,3 +80,42 @@ public extension TherapySetting {
         }
     }
 }
+
+// MARK: Guardrails
+public extension TherapySetting {
```

Comment:
I pulled out the copy into a centralized location so it's easier to follow/change